### PR TITLE
Add recipe for elyra-code-snippet-extension elyra-pipeline-edition-ex…

### PR DESCRIPTION
**Pull request by akchinSTC**
…tension elyra-python-editor-extension elyra-server

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by akchinSTC** [comment]
@lresende - could you +1 yourself as maintainer here thx!

**Review by lresende — APPROVED** [review]

**Comment by akchinSTC** [comment]
@marcelotrevisani - is there anything else we might need with these before merging?

File: recipes/elyra-code-snippet-extension/meta.yaml
Comment (marcelotrevisani) [inline recipes/elyra-code-snippet-extension/meta.yaml:21]:
```suggestion
    - python >=3.6
```

File: recipes/elyra-pipeline-editor-extension/meta.yaml
Comment (marcelotrevisani) [inline recipes/elyra-pipeline-editor-extension/meta.yaml:21]:
```suggestion
    - python >=3.6
```

File: recipes/elyra-python-editor-extension/meta.yaml
Comment (marcelotrevisani) [inline recipes/elyra-python-editor-extension/meta.yaml:21]:
```suggestion
    - python >=3.6
```

File: recipes/elyra-server/meta.yaml
Comment (marcelotrevisani) [inline recipes/elyra-server/meta.yaml:23]:
```suggestion
    - python >=3.6
```

**Comment by marcelotrevisani** [comment]
Just minor things :)
After that we can integrate

**Comment by akchinSTC** [comment]
@marcelotrevisani - all done! thanks for the helping with the merge!

**Review by marcelotrevisani — APPROVED** [review]

**Pull request merged**
